src/mission_companion.cpp

```diff
@@ -480,9 +480,8 @@ bool talk_function::display_and_choose_opts( mission_data &mission_key, const tr
             .viewport_pos( info_offset )
             .viewport_size( info_height )
             .apply( w_info );
-            if( info_offset < mission_text.size() ) {
-                end_line = std::min( info_height, mission_text.size() - info_offset );
-            }
+            end_line = info_offset < mission_text.size() ?
```

Comment:
```suggestion
            end_line = std::min( info_height, mission_text.size() - info_offset );
```

Reply:
The conditional here is a vestige of previous logic, and should be handled by the same check above the invocation to `scrollbar()`.

`end_line` should be set basically every time.